packages/jest-cli/src/reporters/utils.js

```diff
@@ -74,10 +74,8 @@ const relativePath = (config: {rootDir: Path}, testPath: Path) => {
   return {basename, dirname};
 };
 
-const pluralize = (
-  word: string,
-  count: number,
-) => `${count} ${word}${count === 1 ? '' : 's'}`;
+const pluralize = (word: string, count: number) =>
```

Comment:
Can't have it all! Prettier is still an amazing improvement over constant manual formatting and bickering over eslint rules  😄